gammapy/datasets/make.py

```diff
@@ -83,32 +83,55 @@ def sigma_energy_theta(energy, theta, sigma):
 
 
 def make_test_observation_table(observatory_name='HESS', n_obs=10,
-                                datestart=None, dateend=None,
+                                az_range=Angle([0, 360], 'degree'),
+                                alt_range=Angle([45, 90], 'degree'),
+                                date_range=(Time('2010-01-01T00:00:00',
+                                                 format='isot', scale='utc'),
```

Comment:
`format='isot', scale='utc'` is the standard we defined in file:///home/mapaz/astropy/development_code/gammapy/docs/_build/html/time/index.html#time-handling-in-gammapy
It might be the default in Astropy right now (I'm not sure). But if it changes at some point for some reason, we would have to change all entries throughout Gammapy, to be consistent with our standard. Isn't it better to just explicitly give the format and scale?

BTW, what do you mean by `it's important'?